docs(cn): translate docusaurus/docs/title-and-meta-tags.md into Chinese (#41)
docschina/create-react-app: docusaurus/docs/title-and-meta-tags.md

---
id: title-and-meta-tags

> title: 标题与源标签

Reviewer: ```suggestion
title: 标题与元标签
```

> sidebar_label: 标题 & 源标签

Reviewer: ```suggestion
sidebar_label: 标题 & 元标签
```

    </script>
```

> 然后，在服务器上，你可以在发送响应之前，将 `__SERVER_DATA__` 替换为真实数据 JSON。客户端代码可以读取 `window.SERVER_DATA` 以使用它。**请确保[在将 JSON 发送到客户端之前先对其进行消毒](https://medium.com/node-security/the-most-common-xss-vulnerability-in-react-js-applications-2bdffbcc1fa0)，以免你的应用收到 XSS 攻击。**

Reviewer: ```suggestion
然后，在服务器上，你可以在发送响应之前，将 `__SERVER_DATA__` 替换为真实的 JSON 数据。客户端代码可以读取 `window.SERVER_DATA` 以使用它。**请确保 [在将 JSON 发送到客户端之前先对其进行无害化处理](https://medium.com/node-security/the-most-common-xss-vulnerability-in-react-js-applications-2bdffbcc1fa0)，以免你的应用收到 XSS 攻击。**
```
